src/components/commons/Buttons.tsx

```diff
@@ -70,6 +71,43 @@ function ButtonElement({
   );
 }
 
+export function SecondaryButton({
+  text,
+  className,
+  disabled,
+  onClick,
+  href,
+}: {
+  text: string;
+  className?: string;
+  disabled?: boolean;
+  onClick?: () => void;
+  href?: string;
+}) {
+  return (
+    <Link
+      rel="noreferrer"
+      target="_blank"
+      href={href ?? ""}
```

Comment:
hmm what's the point of having `""` as href, should enforce it on the prop level?

Reply:
Wanted to cater for reusability in the future but we can enforce it for now